### DISCUSSION TITLE
Deleting messages in bulk or hotkey?

### DISCUSSION THREAD

**Discussion by capsicumw** [Keriew/augustus#1148] [General]
I play a slow game and end up with hundreds, and even thousands, of alert messages. Most are unread minor alerts so the "delete read messages" button does very little. I feel like my right mouse button is going to wear out deleting them (not to mention the time it takes).   A keyboard hotkey would also work as I could just hold it down and let auto-repeat zip through the list, but I don't find anything like this.

I found some functions in src/window/message_list.c for a button that should delete all non-custom messages "delete_common". Always close to "delete_read" in the source, but I do not have anything like a delete all common (unread) messages button in the message dialog. I do have the delete read messages button, which is basically the opposite of what I want. I only read the keepers that I may want as future reference (price changes, custom events, etc), the unread messages are all petty fire or crime stuff.

Using 4.0.0 appImage on Debian Linux.

Is there a way to clear the messages by modifying the save file as a work around?

Also the messages stop accumulating at 1000(newest saved message is 20 years old), so I have no idea why I suddenly have rubble in part of the city.

**Comment by Keriew** [comment]
Delete common messages was added after the 4.0.0 release. If you want to use it, you'll have to update to the unstable, or wait for the next release.

**Reply by capsicumw** [reply]
Ah ok. 
I must have used the git history wrong, it looked like that code segment was added several years ago. I was probably seeing the file creation date or something like that.
